ENH: Add issue template for scans logging (#219)
TheAxonLab/hcph-sops: docs/data-collection/pre-session.md

    ![nuprep-tube](https://shop.neurospec.com/media/catalog/product/cache/8af976a03d45f9b6d7f91e69feeeffb9/w/e/weaver_nuprep-skin-prep-gel-single-tube-front.jpg)

> - [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session [ON WHICH COMPUTER SHOULD THIS BE DONE?]

Author: I believe this could be done on my laptop or on someone's phone - what do you think ?

Reviewer: I would do it on a laptop rather than a phone, because the screen is bigger so you can have an overview of the form in one glance. It seems easier to type long text as well.
And importantly, it reduces the risk of bringing your phone with you in the MR room ^^''

Author: Indeed, it's a nice additional safety to have :)

Author: ```suggestion
- [ ] Open a new issue in [the github repository](https://github.com/TheAxonLab/hcph-sops/) to collect comments and annotations about the session (a laptop should be available to fill the issue).
```

    - [ ] Under the section `Issues`, click on <span class="consolebutton green">New issue</span>
    - [ ] This should lead you to a page that looks like this 

>         ![](../assets/images/session_issue_template.png)

Author: I had to fix some conflicts and for some reason the image never uploaded (and I can't find anywhere) - @celprov could you please "manually" upload the image to this PR/branch ?
